specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2019-10-27-preview/examples/OpenShiftManagedClustersGet.json

```diff
@@ -0,0 +1,82 @@
+{
+  "parameters": {
+    "api-version": "2019-10-27-preview",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/openShiftManagedClusters/clustername1",
+        "location": "location1",
+        "name": "clustername1",
+        "tags": {
+          "archv2": "",
+          "tier": "production"
+        },
+        "type": "Microsoft.ContainerService/OpenShiftManagedClusters",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "openShiftVersion": "v3.11",
+          "clusterVersion": "osa.1",
+          "fqdn": "df9337748a4b.location1.cloudapp.azure.com",
+          "publicHostname": "openshift.1225bdf9337748a4bd3a.location1.azmosa.io",
```

Comment:
If we do a GET on a private cluster, this will be `10.0.0.254`

Reply:
does it make sense to add a new file `OpenShiftManagedClustersGetPrivateCluster.json`? Because this example as it is now is still a valid example

Reply:
Yea I guess this makes sense. 
We'll also need a new separated one for PUT `OpenShiftManagedClustersPutPrivateCluster `

Reply:
I have added this file in `examples`:
```
specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2019-10-27-preview
├── examples
│   ├── OpenShiftManagedClustersCreate_Update.json
│   ├── OpenShiftManagedClustersDelete.json
│   ├── OpenShiftManagedClustersGet.json
│   ├── OpenShiftManagedClustersListByResourceGroup.json
│   ├── OpenShiftManagedClustersList.json
│   ├── OpenShiftManagedClustersPrivateClusterCreate_Update.json  <--- new
│   └── OpenShiftManagedClustersUpdateTags.json
└── openShiftManagedClusters.json

```